webpages/settings/index.js

```diff
@@ -665,5 +643,7 @@ let fuse;
     }
   });
 
-  chrome.runtime.sendMessage("checkPermissions");
+  sendPermissionCheck();
 })();
+
+checkAndOpenUnsupportedPage();
```

Comment:
Setting pages shouldn't need to call this. Instead, the extension itself should call it.

Reply:
That also means, there's no reason to expose this function to settings pages.